src/codelabs/dart-cheatsheet.md

```diff
@@ -473,19 +458,45 @@ void main() {
   }
 
   if (errs.isEmpty) {
-    _result(true);
+    print('Success!');
   } else {
-    _result(false, errs);
+    errs.forEach(print);
   }
+
+  // ignore_for_file: unnecessary_type_check
 }
-{$ end test.dart $}
-{$ begin hint.txt $}
-This exercise is fairly straightforward.
-Just add a list, set, or map literal after each equals sign.
-See the codelab text for the correct syntax to use.
-{$ end hint.txt $}
+{$ end main.dart $}
 ```
 
+<details markdown="1">
+ <summary>
+   <b>Solution for collection literals example</b>
+  </summary>
+
+  This exercise is fairly straightforward.
+  Just add a list, set, or map literal after each equals sign.
+  See the codelab text for the correct syntax to use.
```

Comment:
I know you didn't write this, but while we're here, can we remove this text that makes it sound easy? It might not be easy for someone.

Also the "see the codelab text..." portion doesn't make sense anymore with the answer right below.

I can't really think of any super valuable hint here, if you can't either, I'd be fine just dropping it. Perhaps we could mention something like "Just remember to specify the types when they can't be inferred." or something like that for the empty ones?

I'll come back to this comment if I think of anything else. Thanks!!

Reply:
I put:

    Add a list, set, or map literal after each equals sign.
    Remember to specify the types for the empty declarations,
    since they can't be inferred.

I think it's ok just as a description of the solution, not necessarily a hint